docs/src/main/paradox/data-transformations/simple-codecs.md

```diff
@@ -29,7 +29,7 @@ is parsed into frames:
   symbol2=PekkoVersion
   value2=$akka.version$
```

Comment:
reference to `akka.version` here and below

Reply:
the build still uses Akka jars - and this param is set in the sbt logic (build.sbt and project/*.scala) - so I think it would easier to wait till we start using pekko dependencies to fix this part